### PR TITLE
add example of using mutate_at() with vars() to summarise_all() docs

### PR DESCRIPTION
**Pull request by benmarwick**
and note about needing to use vars when not quoting variables when this first appears (summarise_at example)

### CODE REVIEW

File: R/colwise-mutate.R [hunk 1]
```diff
@@ -35,9 +35,15 @@
 #' # * _if affects variables selected with a predicate function:
 #'
 #' starwars %>% summarise_at(vars(height:mass), mean, na.rm = TRUE)
+#' Note that vars() is required to quote the variables, or they can be supplied
+#' with quotes
 #' starwars %>% summarise_at(c("height", "mass"), mean, na.rm = TRUE)
 #' starwars %>% summarise_if(is.numeric, mean, na.rm = TRUE)
 #'
+#' mutate_at can transform a selection of variables using regular
```
Comment (lionel-) [inline R/colwise-mutate.R]:
```
#' You can supply selection helpers to _at() functions but you have to quote them with vars():
```

File: R/colwise-mutate.R [hunk 2]
```diff
@@ -35,9 +35,15 @@
 #' # * _if affects variables selected with a predicate function:
 #'
 #' starwars %>% summarise_at(vars(height:mass), mean, na.rm = TRUE)
```
Comment (lionel-) [inline R/colwise-mutate.R]: Could you move that line with the `mutate_at()` example please

File: R/colwise-mutate.R [hunk 3]
```diff
@@ -35,9 +35,15 @@
...
+#' Note that vars() is required to quote the variables, or they can be supplied
```
Comment (lionel-) [inline R/colwise-mutate.R]: Remove this comment.

**Comment by benmarwick** [comment]
Thanks, I've made an update, Did I address your comments properly?

**Comment by lionel-** [comment]
Thanks @benmarwick. We're in the middle of a CRAN release so I'll wait until it's over to merge this.

**Comment by benmarwick** [comment]
No worries, thanks for your help in making this addition

**Comment by lionel-** [comment]
Thanks!

**Pull request merged**